config/services.yaml

```diff
@@ -0,0 +1,201 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    security.exception_listener.class: App\Security\ExceptionListener
+
+    # default params which can be overriden by config/parameters.yaml
+    force_ssl: false
+    forced_ssl_hosts: []
+    aws_metadata: []
+    packagist_metadata_dir: '%kernel.cache_dir%/composer-packages-build'
+    packagist_dumper_compress: 0
+    trusted_ip_header: ''
+    github.webhook_secret: '%env(APP_GITHUB_WEBHOOK_SECRET)%'
+    google_analytics: null
+
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        autowire: true      # Automatically injects dependencies in your services.
+        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+        bind:
+            # services
+            $redis: '@snc_redis.default_client'
+            $redisCache: '@snc_redis.cache_client'
+            $instantMailer: '@swiftmailer.mailer.instant'
+
+            # params
+            $dbUrl: '%env(DATABASE_URL)%'
+            $awsMetadata: '%aws_metadata%'
+            $cacheDir: '%kernel.cache_dir%'
+            $webDir: '%kernel.root_dir%/../web/'
+            $rootDir: '%kernel.root_dir%'
+            $metadataDir: '%packagist_metadata_dir%'
+            $trustedIpHeader: '%trusted_ip_header%'
+            $algoliaIndexName: '%env(ALGOLIA_INDEX_NAME)%'
+
+    # makes classes in src/ available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    App\:
+        resource: '../src/'
+        exclude:
+            - '../src/DependencyInjection/'
+            - '../src/Entity/'
+            - '../src/Kernel.php'
+            - '../src/Tests/'
+
+    # controllers are imported separately to make sure services can be injected
+    # as action arguments even if you don't extend any base controller class
+    App\Controller\:
+        resource: '../src/Controller/'
+        tags: ['controller.service_arguments']
+
+    # add more service definitions when explicit configuration is needed
+    # please note that last definitions always *replace* previous ones
+
+
+#    App\Command\:
+#        resource: '../../Command/*'
+#        tags: ['console.command']
+
+    App\Entity\:
+        resource: '../src/Entity/*Repository.php'
+
+ #   App\Service\:
+ #       resource: '../../Service/*'
+
+#    App\SecurityAdvisory\:
+#        resource: '../../SecurityAdvisory/*.php'
+
+    App\Twig\PackagistExtension:
+        tags:
+            - { name: twig.extension }
```

Comment:
not necessary. The directory loading line 42 already defines such service, and the autoconfiguration is already adding the tag (same for several other services defined below btw)